modules/monitoring-default-monitoring-targets.adoc

```diff
@@ -27,6 +27,7 @@ endif::openshift-dedicated,openshift-rosa[]
 * OpenShift API server
 * OpenShift Controller Manager
 * Operator Lifecycle Manager (OLM)
+* Vector (if Logging is installed)
```

Comment:
@bburt-rh - Would it make sense to add Vector to the glossary further down that preview page since Fluentd as one? Also, the definition for Fluentd is outdated.

Reply:
@libander  - Thanks. Good point. I'm going to write up a separate Jira for making additional changes. In the meantime, I'll go ahead and merge this PR as-is.